src/typescript-worker/build.ts

```diff
@@ -22,9 +23,10 @@ export const build = async (
   },
   emit: (result: BuildOutput) => void
 ): Promise<void> => {
-  const moduleResolver = new ModuleResolver(importMap);
+  const moduleResolver = new ModuleResolver(config.importMap);
+  const cdn = new CachingCdn(config.cdnBaseUrl ?? 'https://unpkg.com/');
```

Comment:
In the readme it mentions that unpkg is used behind the scenes. Is the `cdnBaseUrl` configurable via a public interface such that another cdn could be used?